frontend/frontend.py

```diff
@@ -158,30 +158,30 @@ def draw_gradio_ui(opt, img2img=lambda x: x, txt2img=lambda x: x,imgproc=lambda
                 with gr.Row().style(equal_height=False):
                     with gr.Column():
                         gr.Markdown('#### Img2Img Input')
-                        img2img_image_editor = gr.Image(value=sample_img2img, source="upload", interactive=True,
```

Comment:
Conflict with img2img fixes?

Reply:
I think this was a fix for the img2img as you said, this part is not on the dev branch version of this file, might have to be added manually again.